.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
```

Comment:
Please refer to [this](https://stackoverflow.com/questions/60345326/eslint-no-files-matching-the-pattern-lint-were-found-please-check-for-typing) answer on StackOverflow to fix the error: `No files matching the pattern "."`